Purpose: Adjust the insserv configuration for Debian init.d script names.
Fixes:   None, made as part of Debian porting.
Status:  Debian specific, showed to upstream.
---
--- a/insserv.conf
+++ b/insserv.conf
@@ -1,29 +1,29 @@
 #
 # All local filesystems are mounted (done during boot phase)
 #
-$local_fs	boot.localfs +boot.crypto
+$local_fs	+mountall +mountall-bootclean +mountoverflowtmp +umountfs
 
 #
 # Low level networking (ethernet card)
 #
-$network	network
+$network	+networking +ifupdown
 
 #
 # Named is operational
 #
-$named		+named +dnsmasq +lwresd $network
+$named		+named +dnsmasq +lwresd +bind9 +unbound $network
 
 #
 # All remote filesystems are mounted (note in some cases /usr may
 # be remote. Most applications that care will probably require
 # both $local_fs and $remote_fs)
 #
-$remote_fs	$local_fs +nfs +smbfs
+$remote_fs	$local_fs +mountnfs +mountnfs-bootclean +umountnfs +sendsigs
 
 #
 # System logger is operational
 #
-$syslog		syslog
+$syslog		+rsyslog +sysklogd +syslog-ng +dsyslog +inetutils-syslogd
 
 #
 # SunRPC portmapper available
@@ -33,9 +33,9 @@ $portmap	portmap
 #
 # The system time has been set correctly
 #
-$time		boot.clock +xntpd
+$time		+hwclock
 
 #
 # Services which need to be interactive
 #
-<interactive>   apache apache2 boot.clock boot.crypto boot.crypto-early boot.localfs boot.rootfsck kbd kdump ntp
+<interactive>	glibc udev console-screen keymap keyboard-setup console-setup cryptdisks cryptdisks-early checkfs-loop
